app/app/src/main/java/com/example/clubwat/model/Data.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.clubwat.model
+
+data class Data(
```

Comment:
Nit Pick - Not really important but I think most of the fe is in camel case and this is snake case

Reply:
btw, @SerializedName("") needed to be able to switch these to camel case.